### DISCUSSION TITLE
question about the weather widget

### DISCUSSION THREAD

**Discussion by obtuseoperator** [lezzthanthree/Needy-Streamer-Overload#13] [Q&A] [answered]
i couldn't find a setting for this while i was searching, so i figured id try my luck asking here - is there any way to switch the temperature in the calendar widget from celsius to fahrenheit?

**Comment by RadiantRoar** [comment]
I was also wondering this!

**Comment by lezzthanthree** [comment, marked as answer]
Edit the file on [`Needy Streamer Overload\@Resources\Settings\Calendar\Settings.inc`](https://github.com/lezzthanthree/Needy-Streamer-Overload/blob/master/%40Resources/Settings/Calendar/Settings.inc)

Change the `Units` variable from `metric` to `imperial`

Will make a feature that will allow user to change their measurement system.
